tutorial_src/gue/prog/gue.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _gue_h
+#define _gue_h
+
+int init();
```

Comment:
Dla kompatybilności z czystym C, tu pownno być

int init(void);

W przeciwnym razie w C to oznacza funkcję ze zmienną liczbą argumentów.